### PR TITLE
hardening/892_use_apiary_for_documenting

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #892 
* No tests are required
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-ngsi/api/v1/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
...{\n            \"success\":\"false\",\n            \"error\":\"Invalid subscription, field \'notifyConditions\' is missing\"\n        }\n\n## Group Admin API\n\n### Gets the log4j configuration [GET /admin/log]\n\n+ Request\n\n       GET http://<cygnus_host>:<management_port>/admin/log\n\n+ Response 200 (application/json)\n\n       {\n           \"log4j\": {\n           \"appenders\": [\n               {\n                   \"layout\": \"...\",\n                   \"name\": \"...\"\n               }\n            ],\n            \"level\": \"...\"\n           },\n           \"success\": \"true\"\n       }\n\n+ Response 500 (application/json)\n\n       {\n           \"error\": \"...\"\n       }\n\n### Update the logging level of Cygnus [PUT /admin/log]\n\n+ Parameters\n\n        + loglevel: Must be DEBUG, INFO, WARN, ERROR o FATAL (String) - A valid log level.\n\n+ Request\n\n       PUT http://<cygnus_host>:<management_port>/admin/log?level=<log_level>\n\n+ Response 200 (text/plain)\n\n       OK\n\n+ Response 400 (application/json)\n\n       {\n           \"error\":\"Invalid log level\"\n       }\n\n+ Response 400 (application/json)\n\n       {\n           \"error\":\"\"\n       }\n"
```
Comment (frbattid) [inline cygnus-ngsi/api/v1/index.html]: Why `fiware-ngsi2.apiblueprint.org`? It seems to be something highly coupled to Orion NGSI, not Cygnus.

Comment (pcoello25) [inline cygnus-ngsi/api/v1/index.html]: It seems that is an internal host used by apiary. Apiary use a "mock" server for define the behaviour of the request. I forgot to change that host, next commit will fix it :)

Comment (pcoello25) [inline cygnus-ngsi/api/v1/index.html]: Fixed in 378ccac

**Comment by frbattid** [comment]
LGTM

**Pull request merged**